backend-comparison/src/persistence/base.rs

```diff
@@ -77,10 +77,23 @@ pub fn save<B: Backend>(
             record.results.name, record.results.timestamp
         );
         let file_path = cache_dir.join(file_name);
-        let file = fs::File::create(file_path).expect("Benchmark file should exist or be created");
+        let file =
+            fs::File::create(file_path.clone()).expect("Benchmark file should exist or be created");
         serde_json::to_writer_pretty(file, &record)
             .expect("Benchmark file should be updated with benchmark results");
 
+        // Append the benchmark result filepath in a temp file to be later picked by benchrun
+        let curdir_filepath = env::current_dir()
```

Comment:
Not sure what the current directory is. Let's use the cache directory for the temporary results file.